python-checks/src/main/java/org/sonar/python/checks/AllBranchesAreIdenticalCheck.java

```diff
@@ -53,7 +57,15 @@ private static void handleIfStatement(IfStatement ifStmt, SubscriptionContext ct
     if (!CheckUtils.areEquivalent(body, ifStmt.elseBranch().body())) {
       return;
     }
-    ctx.addIssue(ifStmt.keyword(), "Remove this if statement or edit its code blocks so that they're not all the same.");
+    PreciseIssue issue = ctx.addIssue(ifStmt.keyword(), "Remove this if statement or edit its code blocks so that they're not all the same.");
+    List<Token> ifBodyTokens = TreeUtils.nonWhitespaceTokens(ifStmt.body());
+    issue.secondary(IssueLocation.preciseLocation(ifBodyTokens.get(0), ifBodyTokens.get(ifBodyTokens.size() - 1), null));
+    ifStmt.elifBranches().forEach(e -> {
+      List<Token> elifBodyTokens = TreeUtils.nonWhitespaceTokens(e.body());
+      issue.secondary(IssueLocation.preciseLocation(elifBodyTokens.get(0), elifBodyTokens.get(ifBodyTokens.size() - 1), null));
```

Comment:
`elifBodyTokens.get(ifBodyTokens.size() - 1)` is wrong here, it should be `elifBodyTokens.get(elifBodyTokens.size() - 1)`

Of course, nothing fails because all branches have the same number of tokens as they are identical...

Suggestion: extract a method with a signature such as: `IssueLocation issueLocation(StatementList body)`